target/linux/pistachio/creator-platform-default.config

```diff
@@ -0,0 +1,22 @@
+CONFIG_TARGET_pistachio=y
```

Comment:
Actually we wanted to have version information and downloads server url pointed to our downloads.creatordev.io, until all our packages are available on standard openwrt downloads server.